### ISSUE TITLE
"NEW" icon spills over text when browser window is smaller than available space for text

### ISSUE THREAD

**Issue by Gonzom** [Js41637/Overwatch-Item-Tracker#205]
The "NEW" icon should have some margin between it and the text, as when the browser window is smaller than available space for text, it spills over to the text.

Browser: Chrome.
![newicon](https://user-images.githubusercontent.com/16445220/38675850-c6802ce4-3e61-11e8-8faf-49c7cda6dcde.png)

**Comment by Js41637** [comment]
Yeah I was pretty tired when I added it and really couldn't be bothered lol, i'll be fixing some things today.

**Comment by Gonzom** [comment]
No worries. You asked to check if everything is working so was helping out :)
